content/docs/connect/connect-intro.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Connect to Neon
+subtitle: Everything you need to know about connecting to Neon
+enableTableOfContents: true
+---
+
+Welcome to the connection documentation for Neon. Here, you will find detailed information and instructions for connecting to Neon from different clients and applications, troubleshooting connection issues, connection pooling, and more.
```

Comment:
I wonder about the "Welcome to..." sentence on these landing pages. Consider cutting it out and going right from the subtitle to "Find detailed information..."

Reply:
Good idea! I adjusted all landing pages accordingly.